entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+cd web
```

Comment:
This script won't abort when commands fail and with unbound (a.k.a. undefined) variables (as with the typo below). However, given that some operating systems' `sh` shells seem to not support `set -o pipefail` yes, [even though it should be supported everywhere](https://unix.stackexchange.com/a/654932/593065), I suggest using `bash` instead.

Also, add a shebang, to avoid confusion about which shell to run this file with.

```suggestion
#!/bin/bash
set -eu -o pipefail

cd web
```